src/common/services/api/designations.service.js

```diff
@@ -217,9 +219,9 @@ class DesignationsService {
           }
 
           // Copy jcr:title
-          if (data.data['jcr:content']['jcr:title']) {
-            itemConfig['jcr-title'] = data.data['jcr:content']['jcr:title']
-          }
+          // if (data.data['jcr:content']['jcr:title']) {
```

Comment:
Don't comment these lines of code out, they are necessary for pulling data from AEM and displaying it to the user.